transport_nantes/mobilito/static/mobilito/js/geolocation_map.js

```diff
@@ -110,17 +110,36 @@ function onMapClick(e) {
 }
 map.on('click', onMapClick);
 
+// From django docs
```

Comment:
It's good that you site that it comes from django docs, but you should include a URL.

Reply:
Finally, how is this function different from `request.COOKIES.get()`?  This requires documentation.

Reply:
> It's good that you site that it comes from django docs, but you should include a URL.

I was unsure about it since that answer : 
https://github.com/transport-nantes/tn_web/pull/938#issuecomment-1258399627
> I don't think you need to provide doc links [...]  It's good that you've learned this, but you can generally assume that competent developers know how to look up docs.

> Finally, how is this function different from request.COOKIES.get()? This requires documentation.
request.COOKIES.get is a django function on HttpRequest objects, while this function is usable in Javascript.
It parses document.cookies, which returns a string, to get the cookie that is relevant to me.

Reply:
When you copy code from somewhere, you should absolutely credit from whence you took it.  That is both politeness and honesty.  Explaining in detail what it does would be a different story, since it's obvious in this case.